test/testbool.cpp

```diff
@@ -857,6 +857,16 @@ class TestBool : public TestFixture {
               "}");
         ASSERT_EQUALS("[test.cpp:2]: (style) Incrementing a variable of type 'bool' with postfix operator++ is deprecated by the C++ Standard. You should assign it the value 'true' instead.\n", errout.str());
 
+        check("void f(bool* test){\n"
+              "    (*test)++;\n"
+              "}");
+        ASSERT_EQUALS("[test.cpp:2]: (style) Incrementing a variable of type 'bool' with postfix operator++ is deprecated by the C++ Standard. You should assign it the value 'true' instead.\n", errout.str());
+
+        check("void f(bool* test){\n"
+              "    test[0]++;\n"
+              "}");
+        ASSERT_EQUALS("[test.cpp:2]: (style) Incrementing a variable of type 'bool' with postfix operator++ is deprecated by the C++ Standard. You should assign it the value 'true' instead.\n", errout.str());
+
```

Comment:
If I had to write black box tests I'd probably also come up with `*(&test)`. and `foo+test++` and such, but probably that would be tests for the AST itself.

Reply:
I considered adding more, but arrived at the same conclusion. It seemed right to at least add a test case that was failing before the change, though.